server/services/educationAndWorkPlanApiPersonalLearningPlanService.test.ts

```diff
@@ -0,0 +1,81 @@
+import { aValidActionPlanResponse } from '../data/localMockData/actionPlanResponse'
+import personalLearningPlanActionPlanMapper from '../interfaces/mappers/personalLearningPlanActionPlanMapper'
+import { aValidPersonalLearningPlanActionPlan } from '../data/localMockData/personalLearningPlanActionPlan'
+import EducationAndWorkPlanApiPersonalLearningPlanService from './educationAndWorkPlanApiPersonalLearningPlanService'
+
+jest.mock('../interfaces/mappers/personalLearningPlanActionPlanMapper')
```

Comment:
The diff is showing this as a new class, but actually all I've done is move it out of `personalLearningPlanService.test.ts`